### PR TITLE
LVNOD-30_Config database and server

### PR DESCRIPTION
**Pull request by shyyka**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-XXX)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVNOD-XXX)

## Code reviewers

- [x] @yarvanya
- [x] @yurii-kokhalevych
- [x] @denisbodnar

### Second Level Review

- [x] @lhalam
- [x] @4asof4ik

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo
## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+const bodyParser = require('body-parser');
+
+const app = express();
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+require('./routes')(app);
+app.get('*', (req, res) => res.status(200).send({
+  message: 'Welcome to the API Da-Mi.',
```
Comment (denisbodnar) [inline server/app.js]: Comma not needed

**Review by yarvanya — DISMISSED** [review]

File: config/db.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+const config = require(`${__dirname}/./config.json`)[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable]);
+} else {
+  sequelize = new Sequelize(
+    db_example.database, db_example.username, db_example.password, db_example
+  )
```
Comment (denisbodnar) [inline config/db.js]: need ; at the end

**Review by denisbodnar — DISMISSED** [review]

**Review by yurii-kokhalevych — DISMISSED** [review]

**Review by yarvanya — DISMISSED** [review]

File: server/app.js (hunk 1 above)
Comment (shyyka) [reply server/app.js]: Done.

**Review by yurii-kokhalevych — DISMISSED** [review]

**Review by denisbodnar — DISMISSED** [review]

**Review by yurii-kokhalevych — DISMISSED** [review]

**Review by denisbodnar — DISMISSED** [review]

File: config/db.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const basename = path.basename(module.filename);
+const env = process.env.NODE_ENV || 'development';
+const dbExample = require(`${__dirname}/./db_example.json`)[env];
```
Comment (4asof4ik) [inline config/db.js]: Do you need dbExample ?

File: config/db.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+const config = require(`${__dirname}/./config.json`)[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable]);
+} else {
+  sequelize = new Sequelize(
+    dbExample.database, dbExample.username, dbExample.password, dbExample
```
Comment (4asof4ik) [inline config/db.js]: dbExample?

**Review by yarvanya — DISMISSED** [review]

File: server/app.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+const logger = require('morgan');
+const bodyParser = require('body-parser');
+
+const app = express();
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+require('./routes')(app);
+app.get('*', (req, res) => res.status(200).send({
```
Comment (4asof4ik) [inline server/app.js]: add empty line

**Review by yurii-kokhalevych — DISMISSED** [review]

**Review by yarvanya — DISMISSED** [review]

**Review by denisbodnar — DISMISSED** [review]

**Review by yarvanya — APPROVED** [review]

**Review by denisbodnar — APPROVED** [review]

**Review by yurii-kokhalevych — APPROVED** [review]

**Review by 4asof4ik — APPROVED** [review]

**Pull request merged**